doc/Makefile

```
@@ -7,29 +7,25 @@ SPHINXAUTOGEN = sphinx-autogen
 BUILDDIR      = _build
 
 # Internal variables.
-ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees  $(SPHINXOPTS) .
+ALLSPHINXOPTS = -d $(BUILDDIR)/doctrees  $(SPHINXOPTS) .
 
-.PHONY: help clean html linkcheck doctest api
-
-all: html
+.PHONY: help api html server clean
 
 help:
 	@echo "Please use \`make <target>' where <target> is one of"
 	@echo "  all        generate the complete webpage"
```

```suggestion
	@echo "  all        build the HTML files from the existing rst sources"
```
Might as well be obvious that this does the same thing as html

---

done